### ISSUE TITLE
Base table or view ot found - Have to drop and recreate database

### ISSUE THREAD

**Issue by birdy247** [vierge-noire/cakephp-fixture-factories#180]
I am not entirely sure how to replicate, but from time to time I get a message similar to the below

SQLSTATE[42S02]: Base table or view not found: 1146 Table 'test_ff_project.calendars' doesn't exist

The only way to resolve this is to drop the database and create a new one.  It then works ok

**Comment by pabloelcolombiano** [comment]
Hard to follow. Let me know if you find a way to reproduce.
